bin/format_exercises

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Format existing exercises using rustfmt
+
+EXERCISES_PATH="../exercises/"
+
+# Some exercises use custsom formatting for illustration purpose and should not be formatted. 
+IGNORED_EXERCISES=(
+	"diamond" 
+	"minesweeper" 
+	"rectangles"
+)
+
+cd "$EXERCISES_PATH"
+
+# Iterate over every exercise directory and if it is not ignored - format it
+for exercise_dir in ./*; do
```

Comment:
Why not `for exercise_dir in ${EXERCISES_PATH}/*; do`? That would make it less dependent on the particular location in which this script is being run.